### DISCUSSION TITLE
How to Edit text/html ?

### DISCUSSION THREAD

**Discussion by pawelkontek** [hluk/CopyQ#2856] [Q&A]
How to Edit `text/html` ?

When I click on "Edit" I am only presented with the `text/plain` layer.
I really need to edit the `text/html
![image](https://github.com/user-attachments/assets/05094eb9-dd02-407c-a152-acf47f69007f)

**Comment by hluk** [comment]
You can use the command below which creates a menu entry (shortcut) to edit an HTML format if the current item has it ([here is how to add the command to CopyQ](http://copyq.readthedocs.io/en/latest/faq.html#how-to-load-shared-commands-and-share-them)):

```ini
[Command]
Command="
    copyq:
    editItem(currentItem(), mimeHtml)"
Icon=
InMenu=true
Input=text/html
Name=Edit HTML
Shortcut=ctrl+shift+f2
```
